src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,107 @@
+package duke;
```

Comment:
Can be packaged into smaller parts for easier management (e.g. duke.util.Parser).